src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,12 +6,15 @@
 public class CliSyntax {
 
     /* Prefix definitions */
-    public static final Prefix PREFIX_NAME = new Prefix("n/");
```

Comment:
need to leave this for the time being so that the original commands don't break, until we get to removing them